diff -up kcharselect-19.08.2/kcharselectdia.cc.1~ kcharselect-19.08.2/kcharselectdia.cc
--- kcharselect-19.08.2/kcharselectdia.cc.1~	2019-10-06 05:09:04.000000000 +0200
+++ kcharselect-19.08.2/kcharselectdia.cc	2019-10-14 14:28:56.050232787 +0200
@@ -265,7 +265,6 @@ void KCharSelectDia::toClipHTML()
   QClipboard *cb = QApplication::clipboard();
   QString input;
   QString html;
-  QString tempstring;
   QChar   tempchar;
   int i = 0;
 
@@ -282,7 +281,7 @@ void KCharSelectDia::toClipHTML()
         }
       else
         {
-          html.append(tempstring.sprintf("&#x%x;", tempchar.unicode()));
+          html.append(QString::asprintf("&#x%x;", tempchar.unicode()));
         }
     }
   cb->setText(html);
